packages/odyssey-eslint/package.json

```diff
@@ -8,9 +8,12 @@
   "dependencies": {
     "@typescript-eslint/eslint-plugin": "^4.26.0",
     "@typescript-eslint/parser": "^4.26.0",
-    "eslint": "^7.27.0",
+    "eslint": "^7.32.0",
+    "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-header": "^3.1.1",
     "eslint-plugin-jsx-a11y": "^6.4.1",
-    "eslint-plugin-react-hooks": "^4.2.0"
+    "eslint-plugin-prettier": "^4.0.0",
+    "eslint-plugin-react-hooks": "^4.2.0",
+    "prettier": "^2.4.0"
```

Comment:
is this specified as a peer dependency by the eslint packages? Curious why we need to list it as a top level dep ourselves

Reply:
good catch, I dont believe its necessary. removing now.